### ISSUE TITLE
Checkbox component causes a Level A WCAG 4.1.2 error: Ensure every ARIA toggle field has an accessible name (#headlessui-control-_r_l_)

### ISSUE THREAD

**Issue by mmafe** [ccswbs/uofg-components#52]
When testing the checkbox, Accessibility Insights reports a Level A accessibility error.  Oddly the text of the checkbox is visible to my screen-reader, so we'd need to confirm if this is a false positive.

--

Title: WCAG 4.1.2: Ensure every ARIA toggle field has an accessible name (#headlessui-control-_r_l_)
Tags: Accessibility, WCAG 4.1.2, aria-toggle-field-name

Issue: Ensure every ARIA toggle field has an accessible name (aria-toggle-field-name - https://accessibilityinsights.io/info-examples/web/aria-toggle-field-name)

Target application: College of Social and Applied Human Sciences People | University of Guelph - https://tq-next.netlify.app/csahs/people

Element path: #headlessui-control-_r_l_

Snippet: <span class="uofg-checkbox group flex size-4 items-center justify-center rounded border bg-white p-3 transition-colors focus-visible:ring-2 focus-visible:ring-offset-2 focus-visible:outline-none ui-checked:bg-yellow focus-visible:ring-yellow" id="headlessui-control-_r_l_" role="checkbox" aria-checked="false" tabindex="0" data-headlessui-state="">

How to fix: 
Fix any of the following:
  Element does not have text that is visible to screen readers
  aria-label attribute does not exist or is empty
  aria-labelledby attribute does not exist, references elements that do not exist or references elements that are empty
  Element has no title attribute

Environment: Chrome version 147.0.0.0

====

This accessibility issue was found using Accessibility Insights for Web 2.46.0 (axe-core 4.10.2), a tool that helps find and fix accessibility issues. Get more information & download this tool at http://aka.ms/AccessibilityInsights.
